clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/Metadata.scala

```diff
@@ -106,13 +106,12 @@ object Metadata {
     * extension. Rather than make dangerous guesses, we require that the code calling
     * the move operation provide the file extension to retain during the move operation.
     */
-  def findNewPathForMove(
+  def buildFilePath(
     source: URI,
     destination: URI,
     extension: String,
     newBasename: Option[String] = None
   ): URI = {
-    // TODO: Rewrite this using a Path-based API.
```

Comment:
`Paths.get` of a cloud `URI` will explode without the right global NIO config options. Since we're using the SDK instead of NIO, I don't think we'll be reworking this to use `Path`.